### ISSUE TITLE
Could not find installation path for: yaml while switching to PHP 5.6

### ISSUE THREAD

**Issue by diazwatson** [weprovide/valet-plus#376]
```
$ valet use 5.6
Caution! The PHP version you're switching to is EOL.
Please check http://php.net/supported-versions.php for more information.
[php@7.1] Unlinking
Unlinking /usr/local/Cellar/valet-php@7.1/7.1.26... 25 symlinks removed

[libjpeg] Relinking
[php@5.6] Linking
Linking /usr/local/Cellar/valet-php@5.6/5.6.40... 25 symlinks created

If you need to have this software first in your PATH instead consider running:
  echo 'export PATH="/usr/local/opt/valet-php@5.6/bin:$PATH"' >> ~/.zshrc
  echo 'export PATH="/usr/local/opt/valet-php@5.6/sbin:$PATH"' >> ~/.zshrc

[valet-php@5.6] Stopping
[valet-php@7.1] Stopping
[valet-php@7.2] Stopping
[BREW TAP] henkrehorst/php already installed
[PECL] Updating PECL channel: pecl.php.net
[PECL] Installing extensions
	apcu is already installed, skipping...
	apcu successfully enabled
	geoip successfully installed
	geoip successfully enabled

In Pecl.php line 147:

  Could not find installation path for: yaml

  pecl/yaml requires PHP (version >= 7.0.0-dev), installed version is 5.6.40
  No valid packages found
  install failed


use [<service>] [<targetVersion>]
```

**Comment by diazwatson** [comment]
Fixed by installing yaml manually 

```
pecl install yaml-1.3.1
```
After switch to any other working version:
```
valet use 7.1
```
and back to 5.6 to make sure everything works as expected:
```
valet use 5.6
```

**Note:** if you get a message saying there is no php version linked, trying unlinking and linking as follow:
```
brew unlink valet-php@5.6 && brew link --force valet-php@5.6
```

**Comment by SanderAtom** [comment]
I think you could've also run `valet fix` after switching to 5.6, but `yaml` install needs to be looked at since it's giving more install problems.

**Comment by diazwatson** [comment]
I think I tried with `valet fix` but didn't fix the issue

**Comment by cislan-devteam** [comment]
You have to install `libyaml`(brew)  and `yaml` (pecl)

`brew install libyaml`
`pecl install yaml`

**Comment by henkrehorst** [comment]
I added libyaml to the [brewtap](https://github.com/henkrehorst/homebrew-php) as required during an installation of a php version

**Comment by Neodork** [comment]
@diazwatson Could you check the libyaml requirement again?

**Comment by mischabraam** [comment]
Resolved in PR #390
